docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -18,23 +18,40 @@ To make your validator node and validator fullnode deployment hassle-free, make
 ## Hardware requirements
 
 For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
+For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
+TPS on mainnet.
 
-- **CPU**:
-  - 8 cores, 16 threads
-  - 2.8GHz, or faster
-  - Intel Xeon Skylake or newer
-- **Memory**: 32GB RAM.
-- **Storage**: 2T SSD with at least 40K IOPS and 200MiB/s bandwidth.
-- **Networking bandwidth**: 1Gbps
+In order to evaluate that, you can use the following commands
+
+```
+# checkout repo and install needed tools (cargo, etc)
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core/
+./scripts/dev_setup.sh
```

Comment:
Can we refer back to already existing documentation on checking out the aptos-core?

It's copied around multiple places.

Reply:
Done